src/components/Header/Header.jsx

```diff
@@ -44,7 +44,7 @@ function Header() {
       <div className="flex justify-between md:justify-evenly sticky top-0 z-[999] backdrop-blur-sm items-center py-[1.5rem] px-4 md:px-10 ">
         <div className="hover:scale-105 duration-100 ease-in-out">
           <Link to="/">
-            <img width={70} src="\assets\icon.png" alt="Logo" />
+            <img width={260} src="public/assets/Nacto-Care-Logo/Nacto-CARE LOGO No BG.svg" alt="Logo" />
```

Comment:
Filename is wrong
There is no file named as `Nacto-CARE LOGO No BG.svg` in your PR.

The name of file uploaded by you is as below

![image](https://github.com/Nactore-Org/Nacto-Care/assets/79367883/5280f7ad-33bc-4a2d-b274-a6bd4f9871aa)

I am not sure why even with wrong filename your localhost shows the logo. Probably the file name was changed but was not pushed ?